Rlabkey/man/labkey.experiment.saveRuns.Rd

```diff
@@ -0,0 +1,59 @@
+\name{labkey.experiment.saveRuns}
+\alias{labkey.experiment.saveRuns}
+\title{Saves Runs.}
+\description{
+Saves experiment runs.
+}
+\usage{
+labkey.experiment.saveRuns(baseUrl=NULL, folderPath,
+    protocolName, runList)
+}
+\arguments{
+  \item{baseUrl}{(optional) a string specifying the \code{baseUrl} for the labkey server}
+  \item{folderPath}{a string specifying the \code{folderPath} }
+  \item{protocolName}{ a string specifying the protocol name of the protocol to use}
+  \item{runList}{a list of experiment run objects}
+}
+\details{
+Saves experiment runs. Runs may refer to existing data and material objects, either inputs or outputs,
+by ID or LSID. Runs may also define new data and materials objects by not specifying an ID or LSID in their properties.
+
+Refer to the labkey.experiment.createData, labkey.experiment.createMaterial, and labkey.experiment.createRun helper functions
+to assemble the data structure that saveRuns expects.
+}
+
+\value{
+ Returns the object representation of the experiment run.
+}
+
+\author{Ankur Juneja}
+\seealso{
+\code{\link{labkey.experiment.createData}},
+\code{\link{labkey.experiment.createMaterial}},
+\code{\link{labkey.experiment.createRun}}
+}
+\examples{
+\donttest{
+
+library(Rlabkey)
+
+## example with datarows
+
+df <- data.frame(participantId=c(1:3), visitId = c(10,20,30), sex = c("f", "m", "f"))
+run <- labkey.experiment.createRun(list(name="new assay run"), dataRows = df)
+labkey.experiment.saveRuns(baseUrl="http://labkey/", folderPath="home",
+    assayConfig=list(assayName="GPAT", providerName="General"), runList=run)
```

Comment:
See my last email. It sounds like maybe this example needs to also be removed? Either that or we need updated info in the parameters section of this doc as to what params to use to upload an assay run.